_posts/2018-11-12-wdc-supported-lib.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Support for WDC Libraries to Change in Tableau 2019.1"
+abstract: "Web Data Connectors that use older versions of the WDC client library will display a warning in Tableau 2019.1. When Tableau 2019.2 is released, those WDCs will not run unless they have been updated."
+---
+
+Web Data Connectors (WDC) that use older versions of the WDC client library will display a warning in Tableau 2019.1.
+
+`Warning - This version of the Web Data Connector library is deprecated...`
+
+If you are using a Web Data Connector and see this warning message, contact the developer of the connector directly. When Tableau 2019.2 is released, a WDC that uses one of the older libraries will not run, unless the developer updates the WDC to use a supported version of the library.
```

Comment:
Should we say one of the below specified libraries rather than older libraries?

Reply:
Can we say "that users a library prior to 2.0.2"?